src/services/dashboards/resource.js

```diff
@@ -305,7 +305,7 @@ const switchToTopicCentric = (input) => {
               }
               return [...reports, report];
             }, output[topic].reports);
-            output[topic].resources = (resourceObjects || []).reduce((resources, resource) => {
```

Comment:
This branch can be removed. We only end up at this evaluation **because resourceObjects exists and is an array that we are currently iterating over**.